readthedocs/search/parsers.py

```diff
@@ -188,6 +238,9 @@ def _clean_body(self, body):
             body.css('[role=search]'),
             # Permalinks
             body.css('.headerlink'),
+            # Sphinx code blocks contain line numbers and they are very noisy in contents
```

Comment:
We are doing this for all doctools, not just Sphinx.

```suggestion
            # Line numbers from code blocks, they are very noisy in contents.
            # This convention is popular in Sphinx.
```

Reply:
Is it pygments that has this line numbering pattern? Or it's a more widespread thing?